locksmith/src/controllers/userController.ts

```diff
@@ -91,6 +100,45 @@ export const retrieveEncryptedPrivatekey = async (
   }
 }
 
+export const retrieveWaasUuid = async (
+  req: Request,
+  res: Response
+): Promise<any> => {
+  const apiKeyName = process.env.COINBASE_CLOUD_API_KEY_NAME
```

Comment:
Can we do that in the config file like the rest of the env vars please?

Reply:
please use better names: `coinbaseCloudApiKeyName` and `coinbaseCloudPrivateKey`.